dev_block ✊ 
I ran into an error during the last step of running `docker-compose build`
```
 => ERROR [ 9/10] RUN npm install                                                              32.8s
------
 > [ 9/10] RUN npm install:
#0 0.904 npm WARN npm npm does not support Node.js v10.24.0
#0 0.904 npm WARN npm You should probably upgrade to a newer version of node as we
#0 0.904 npm WARN npm can't make any promises that npm will work with this version.
#0 0.904 npm WARN npm Supported releases of Node.js are the latest release of 4, 6, 7, 8, 9.
#0 0.904 npm WARN npm You can find the latest version at https://nodejs.org/
#0 1.058 npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
#0 2.130 npm WARN notice [SECURITY] css-what has the following vulnerability: 1 high. Go here for more details: https://github.com/advisories?query=css-what - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
#0 2.414 npm WARN tar write after end
#0 2.514 npm WARN tar write after end
#0 2.939 npm WARN notice [SECURITY] postcss has the following vulnerability: 1 moderate. Go here for more details: https://github.com/advisories?query=postcss - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
#0 2.998 npm WARN tar write after end
#0 3.055 npm WARN tar write after end
#0 3.389 npm WARN tar write after end
#0 3.695 npm WARN notice [SECURITY] ws has the following vulnerability: 1 moderate. Go here for more details: https://github.com/advisories?query=ws - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
#0 3.759 npm WARN tar write after end
#0 4.715 npm WARN tar write after end
#0 4.817 npm WARN tar write after end
#0 5.427 npm WARN tar write after end
#0 5.511 npm WARN tar write after end
#0 5.829 npm WARN tar write after end
#0 5.894 npm WARN tar write after end
#0 5.943 npm WARN tar write after end
#0 6.283 npm WARN tar write after end
#0 6.583 npm WARN tar write after end
#0 7.533 npm WARN tar write after end
#0 7.780 npm WARN tar write after end
#0 8.576 npm WARN tar write after end
#0 9.195 npm WARN tar write after end
#0 9.351 npm WARN tar write after end
#0 9.584 npm WARN tar write after end
#0 9.738 npm WARN tar write after end
#0 9.899 npm WARN tar write after end
#0 10.18 npm WARN tar write after end
#0 10.50 npm WARN tar write after end
#0 10.61 npm WARN notice [SECURITY] nth-check has the following vulnerability: 1 high. Go here for more details: https://github.com/advisories?query=nth-check - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
#0 10.62 npm WARN notice [SECURITY] normalize-url has the following vulnerability: 1 high. Go here for more details: https://github.com/advisories?query=normalize-url - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
#0 10.79 npm WARN notice [SECURITY] browserslist has the following vulnerability: 1 moderate. Go here for more details: https://github.com/advisories?query=browserslist - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
#0 10.86 npm WARN notice [SECURITY] nanoid has the following vulnerability: 1 moderate. Go here for more details: https://github.com/advisories?query=nanoid - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
#0 10.87 npm WARN tar write after end
#0 11.13 npm WARN notice [SECURITY] minimatch has the following vulnerability: 1 high. Go here for more details: https://github.com/advisories?query=minimatch - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
#0 11.18 npm WARN tar write after end
#0 11.23 npm WARN tar write after end
#0 11.78 npm WARN tar write after end
#0 11.94 npm WARN tar write after end
#0 11.98 npm WARN tar write after end
#0 12.00 npm WARN tar write after end
#0 12.05 npm WARN tar write after end
#0 32.75 npm ERR! cb() never called!
#0 32.75
#0 32.75 npm ERR! This is an error with npm itself. Please report this error at:
#0 32.75 npm ERR!     <https://github.com/npm/npm/issues>
#0 32.77
#0 32.77 npm ERR! A complete log of this run can be found in:
#0 32.77 npm ERR!     /root/.npm/_logs/2023-02-08T22_34_19_168Z-debug.log
------
failed to solve: executor failed running [/bin/sh -c npm install]: exit code: 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM ruby:2.7.3
+FROM ruby:latest
 
 RUN apt-get update
 RUN apt-get install -y \

